workflows/employment_verification.go

```diff
@@ -0,0 +1,86 @@
+package workflows
+
+import (
+	"math/rand"
+	"time"
+
+	"go.temporal.io/sdk/workflow"
+
+	"github.com/temporalio/background-checks/config"
+	"github.com/temporalio/background-checks/signals"
+	"github.com/temporalio/background-checks/types"
+)
+
+func chooseResearcher(ctx workflow.Context, input types.EmploymentVerificationWorkflowInput) (string, error) {
+	researchers := []string{
+		"researcher1@example.com",
+		"researcher2@example.com",
+		"researcher3@example.com",
+	}
+
+	// Here we just pick a random researcher.
+	// In a real use case you may round-robin, decide based on price or current workload,
+	// or fetch a researcher from a third party API.
+
+	researcher := researchers[rand.Intn(len(researchers))]
```

Comment:
Non-deterministic since it's seeded from the outside (technically you could use a custom-seeded random per workflow or put this in a side effect). Technically it probably doesn't matter because it only affects activity input, but we should strive to be deterministic even in our activity params.